runtime/src/launch/amudprun/launch-amudprun.c

```diff
@@ -57,7 +57,7 @@ static void add_env_options(int* argc, char** argv[]) {
   //
   // Duplicate the old argv into the start of the new one.
   //
-  memcpy(new_argv, (*argv), *argc * sizeof((*argv)[0]));
+  chpl_memove(new_argv, (*argv), *argc * sizeof((*argv)[0]));
```

Comment:
Yeah, I think all the launcher ones could be memcpy, but performance doesn't matter for the launchers so I just went with memmove.

Reply:
It's not clear whether you're going to turn it back into a `memcpy()` again but if you weren't I'd reconsider.  A future maintainer could see this `memove()` and puzzle over why it was necessary for a long time, before discovering that it wasn't.  If we're going to differentiate we should try pretty hard to use the right thing in each case, rather than the expedient one.
If you do keep it it's got a typo: it should be `memmove()` (3 m's total).